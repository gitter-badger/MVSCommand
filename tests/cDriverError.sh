(export STEPLIB=CBC.SCCNCMP; mvscmd --pgm="CCNDRVR" --args="/CXX" --syslin=ibmuser.test.obj\(err\) --sysin=ibmuser.test.c\(err\) --sysprint=*)
