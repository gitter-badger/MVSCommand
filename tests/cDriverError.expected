"//'IBMUSER.TEST.C(ERR)'", line 2.2: CCN5859 (S) #error directive: "This program has an error".
CCN0793(I) Compilation failed for file //'IBMUSER.TEST.C(ERR)'.  Object file not created.
